tests/safe_command/test_injection.py

```diff
@@ -0,0 +1,674 @@
+import pytest
+import subprocess
+from pathlib import Path
+from os import mkfifo, symlink, remove, getenv
+from shutil import rmtree, which
+
+from security import safe_command
+from security.safe_command.api import _parse_command, _resolve_paths_in_parsed_command, _shell_expand
+from security.exceptions import SecurityException
+
+with (Path(__file__).parent / "fuzzdb" / "command-injection-template.txt").open() as f:
+    FUZZDB_OS_COMMAND_INJECTION_PAYLOADS = [line.replace('\\n','\n').replace("\\'", "'")[:-1] for line in f] # Remove newline
```

Comment:
Here and below can you just use `rstrip` to remove the newline?

Reply:
No because `rstrip` would remove characters from in the payload as well. I wrote it this way so the payload list can be modified with any payload ending with `\n\t ` etc and only the final newline separating payloads is removed.